roles/bridge/defaults/main.yml

```diff
@@ -3,21 +3,49 @@
 bridge_home_dir: /opt/{{ service_identifier }}
 
 bridge_bin_dir: "{{ bridge_home_dir }}/bin"
-bridge_config_dir: "{{ bridge_home_dir }}/config"
+bridge_blocks_dir: "{{ bridge_home_dir }}/blocks"
 bridge_data_dir: "{{ bridge_home_dir }}/data"
+bridge_index_dir: "{{ bridge_home_dir }}/index"
+bridge_inverted_index_dir: "{{ bridge_home_dir }}/inverted_index"
+bridge_key_dir: "{{ bridge_home_dir }}/keys"
+bridge_keyring_dir: "{{ bridge_key_dir }}/keyring-test"
 bridge_log_dir: "{{ bridge_home_dir }}/log"
```

Comment:
What about transients directory ? did we leave it on purpose because its empty ?

Reply:
it will create it self when started